analyzers/tests/SonarAnalyzer.Test/TestCases/MethodParameterUnused.RoslynCfg.Fixed.cs

```diff
@@ -645,4 +645,18 @@ private static class AssemblyResolver
             }
         }
     }
+
+    // https://github.com/SonarSource/sonar-dotnet/issues/8187
+    public class Repro_8187_CodeFix
+    {
+        void Method()
+        {
+            DoSomething(21);
+        }
+
+        void DoSomething(int a) // Fixed
+        {
+            Console.WriteLine(a);
+        }
+    }
```

Comment:
Add a test with named arguments, which are out of order. Also some tests with optional arguments.